mapbox/libandroid-telemetry/build.gradle

```diff
@@ -32,11 +32,21 @@ dependencies {
     // OkHttp for network requests
     compile rootProject.ext.dep.okhttp3
 
+    // Google Play Services
+    provided rootProject.ext.dep.gmsLocation
+
+    // LOST
+    provided(rootProject.ext.dep.lost) {
+        exclude group: 'com.google.guava'
```

Comment:
Same here: let's update to the latest version. If the latest version is an issue to the developer, now they have a mechanism to switch to a different provider.

Reply:
👀 https://github.com/mapbox/mapbox-java/pull/502#discussion_r142348095